components/language-switcher/language-switcher.html

```diff
@@ -0,0 +1,35 @@
+{# @select
+#
+# parameters:
+# id            - string, required
+# name          - string, required
+# label         - string, required
+# class         - string
+#}
+<form class="mzp-c-language-switcher {% if class %}{{ class }}{% endif %}" method="get" action="#">
+  <a class="mzp-c-language-switcher-link" href="https://www.mozilla.org/locales/" target="_blank" external="noopener noreferrer">Language</a>
+  <label for="mzp-c-language-switcher-select">Language</label>
+  <select id="mzp-c-language-switcher-select" class="mzp-js-language-switcher-select" name="{{ name }}">
```

Comment:
I think this could be an instance of the regular form select, and can even reuse the same context data:

`{% render '@select', { id: 'mzp-c-language-switcher-select', class: 'mzp-js-language-switcher-select', name: 'language', label: 'Language', options: '@select.options' } %}`

If you do that I don't think this component needs a config file at all. Also the parameters above would be reduced just to `class`. Though thinking about it, I'm not sure this component needs an additional class. It doesn't have any variants so any additional classes would be some kind of custom thing. So maybe better to remove that for simplicity.